src/Resource/AbstractResourceList.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PCF\DisposableEmail\Resource;
+
+/**
+ * Class AbstractResourceList
+ * @package PCF\DisposableEmail\Resource
+ */
+abstract class AbstractResourceList
+{
+    /**
+     * @var string
+     */
+    protected static $list = '';
+
+    /**
+     * @return array
+     */
+    public static function getList(): array
+    {
+        return explode(PHP_EOL, trim(static::$list));
```

Comment:
topic to discuss: You can copy list from [source](https://github.com/martenson/disposable-email-domains) as a text. But in other hands, thi lib will be more efficient if we would allready keep list as array